### ISSUE TITLE
Komentář

### ISSUE THREAD

**Issue by Amatthe** [KatPetraskova/PROJEKT-Grid-blog#1]
Ahoj,
hodil jsem na to zatím ještě ne úplně trénovaným očkem a zdá se mi, že by to šlo ještě trochu vypilovat, co se rozmístění obrázků a těch tipů týče. Koukám na to na normálním notebooku a přijde mi, že obrázky mají sklony vytékat z gridu. Taky mě trošku zaskočil ten double grid v article/header.
Zítra si to zkusím vypracovat sám, a snad ti pak budu moct přesněji říct, co a jak.

**Comment by KatPetraskova** [comment]
Jo, ten double grid vysvětlím - mně totiž ten velký grid nefungoval (kvůli tomu obrázku vlevo v tipu - mlsani4.jpg - byl moc velký a nechtěl se smrsknout), tak jsem nasadila extra grid jen na header a v headeru to fungovalo. A když jsem pak přišla na to, že musím nejdřív ten obrázek zmenšit, tak jsem na ten zbytečný grid v headeru zapomněla :)
A rovnou se přiznám, že jsem si vlastně zapomněla zkontrolovat, jestli ty obrázky nepřetékají z gridu. Protože když jsem tam zapásila s tím obrázkem mlsani4.jpg, tak jsem byla pak ve výsledku ráda vůbec za to, jak to nakonec dopadlo :))

**Comment by Amatthe** [comment]
Tak už jsem si to zkusil sám, a přišel jsem na tyhle tipy:
- část problémů řeší nastavení šířky kontejneru (.blog), dal jsem max-width: 1100px a margin auto, aby se hezky zarovnal doprostřed stránky;
- u obrázků řeší hodně max-width: 100% a vynulování marginů prvku figure.

Jinak je to myslím dobrý, jen text u tipů má být zarovnaný vpravo/vlevo, ne doprostřed.

**Comment by KatPetraskova** [comment]
Jo tak, já právě myslela, že se šířkou kontejneru nemáme nic dělat (nebylo explicitně řečeno, že můžeme nastavit jinou velikost). Ale super, děkuju za opravu.
Obrázkům jsem tedy následně řekla, jaká bude šířka (max-width).
A texty u tipů jsem taky srovnala.

A přišla jsem na to, že mi nefunguje celkový velký grid i na header, resp. nadpis h1. Ve Firefoxu (když mám jen grid na .blog a nikde jinde), tak mi to říká, že nadpis h1 nebude začínat v prostředním sloupci, protože jeho rodič (=header) není grid. Takže jsem to sice smazala, ale pak jsem si tam hezky zas napsala, že header je taky grid... Tak nevím.. Možná Luděk v nějaké své přednášce zmiňoval, že grid platí jen na přímé potomky (?), ale fakt si nejsem jistá.

Nicméně jsem opravila, co se dalo :) Děkuju za rady :)

**Comment by Amatthe** [comment]
Málokterý designér ti nalajnuje vždycky úplně všecko. Něco musíš odhadnout. A to umisťování obsahu do kontejneru je hodně obvyklá praxe, protože pak nemusíš extra řešit vzhled na ultraširokých monitorech, televizích a podobných obrazovkách.
